scaffolder-skeletons/catalog-info-skeleton/catalog-info.yaml

```diff
@@ -21,8 +25,14 @@ metadata:
     github.com/project-slug: ${{ values.githubOrg }}/${{ values.repoName }}
 spec:
   type: ${{ values.applicationType }}
+  {%- if values.system %}
   system: ${{ values.system }}
+  {%- endif %}
+  {%- if values.lifecycle %}
+  lifecycle: ${{ values.lifecycle }}
```

Comment:
allowing to specify custom lifecycle, and using `production` as default for backward compatibility